samples/guide/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'kotlin-kapt'
 
 dependencies {
   implementation project(':okhttp')
+  implementation deps.junit
```

Comment:
I’d prefer a solution that doesn’t require downstream projects to change their Gradle configs!